.github/workflows/test-remove-checkout-credentials.yaml

```diff
@@ -0,0 +1,45 @@
+name: Test login-to-gar action
```

Comment:
```suggestion
name: Test remove-checkout-credentials
```

Reply:
LGTM otherwise! Not approving yet to make sure we won't mess up with action test names 😁

Reply:
Fixed, thanks :)